### DISCUSSION TITLE
Welcome to telegram-pdf-bot Discussions!

### DISCUSSION THREAD

**Discussion by MrBotDeveloper** [MrBotDeveloper/telegram-pdf-bot#58] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by maroon-bells** [comment]
Hello! I really liked this project, thank you making it public. But there are some issues to run. Please, can you answer to this questions:
1. What is version of Ubuntu OS?
2. I can not fix this issue  Package 'libgdk-pixbuf2.0-0' has no installation candidate
